docs/contributing.rst

```diff
@@ -267,6 +267,17 @@ Please:
 .. _PEP 8 - Style Guide for Python Code:
   https://www.python.org/dev/peps/pep-0008
 
+Submitting a pull request
+=========================
+Steps:
+
+1. Write your code!
+2. Make sure your environment is set up properly and that you're in your virtualenv (this is a **very important** step).
+3. Run ``./pep8.travis.sh`` to do a cursory check of your code style. Fix any errors.
+4. Run ``tox -e lint`` to check for pylint errors. Fix any errors.
+5. Run ``tox`` to run the unit tests. Fix any errors.
+6. Run the integration tests, see `integration`_.
```

Comment:
I personally don't run them (it's just too complicated to set up) and would mark them here as optional. However, you should have a step "8", which tells you to look at the Travis build status and logs in case of any failures.